scikit/notebooks/Scikit Learn on CMLE.ipynb

```diff
@@ -0,0 +1,702 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Online Prediction with Scikit-Learn on Google Cloud Machine Learning Engine\n",
+    "Alright, so you’ve built your machine learning models with scikit-learn, you want to serve your models in real time for a real application, but managing infrastructure sounds like a nightmare. Thankfully, Google now has the answer. Google supports serving your trained models on Cloud Machine Learning Engine (ML Engine) with online prediction.\n",
+    "\n",
+    "The gist of it is: use a model you already have and use ML Engine’s online prediction service to support scalable prediction requests against your trained model.\n",
+    "\n",
+    "This notebook use the [Census Income Data Set](https://archive.ics.uci.edu/ml/datasets/Census+Income) to create a simple model, train the model, upload the model to ML Engine, and lastly use the model to make predictions. "
```

Comment:
Here you are saying "you will create, train and upload" and above you say get your model to make scalable predictions. So there is bit confusion on the goals.

Reply:
Agreed, removed the first two lines, and just kept "This notebook ... "